docs/content/tutorials/create-environment/index.md

```diff
@@ -0,0 +1,169 @@
+---
+type: docs
+title: "4. Create Environment"
+linkTitle: "4. Create Environment"
+description: "Create a Radius Environment to deploy the application"
+weight: 400
+---
+
+In part four of this tutorial, you will create an environment to register the Recipe for the PostgreSQL resource type.
+
+## Create a workspace
+
+Radius Workspace is a combination of a Kubernetes context, a Radius Environment, and a Resource Group.
+
+Create a new workspace using the [`rad workspace create`]({{< ref rad_workspace_create >}}) command to manage the cluster configuration.
+
+  ```bash
+  rad workspace create kubernetes my-workspace
+  ```
+
+  ```
+  Creating workspace...
+  Set "my-workspace" as current workspace.
+  ```
+
+  Show the current workspace. The `--output json` will show the details of the workspace created.
+
+  ```bash
+  rad workspace show -o json   
+  ```
+
+  ```  
+  {
+    "connection": {
+      "context": "my-kube-context",
+      "kind": "kubernetes"
+    }
+  }
+  ```
+
+## Create a Radius Resource Group
+
+Radius Resource group define the permission boundary for Radius resources.
+
+Create a Radius Resource Group using the [rad group create]({{< ref rad_group_create >}}) command.
+
+```bash
+rad group create my-group
+```
+```
+creating resource group "my-group" in workspace "my-workspace"...
+
+resource group "my-group" created
+```
+
+## Create a Radius Environment
+
+Radius Environment is the landing zone for Radius applications.
```

Comment:
Link to the environments overview page as well: https://docs.radapp.io/guides/deploy-apps/environments/overview/


```suggestion
A Radius Environment is the landing zone for the developer that is preconfigured with all the necessary infrastructure Recipes required to deploy Radius applications.
```